docs/concepts/ipfs-gateway.md

```diff
@@ -50,23 +65,19 @@ A gateway behind a firewall represents just one potential location for a private
 Public gateway operators include:
 
 - Protocol Labs, which deploys the public gateway `https://ipfs.io`.
-- Third-party public gateways. E.g., `https://cf-ipfs.com`.
+- Third-party public gateways such as `https://cf-ipfs.com`.
 
 Protocol Labs maintains a [list of public gateways](https://ipfs.github.io/public-gateway-checker/) and their status.
 
-![A list of public gateways and their status, available on IPFS](./images/ipfs-gateways/public-gateway-checker.png)
-
 ## Gateway types
 
-Categorizing gateways involves several dimensions:
+There are multiple gateway types, each with specific use case, security, performance, and functional implications.
 
 - [Read/write support](#read-only-and-writeable-gateways)
 - [Authentication support](#authenticated-gateways)
 - [Resolution style](#resolution-style)
```

Comment:
In this introduction, I'd add a red warning for **path resolution** because it does not provide origin isolation.